docs/pages/management/guides/joining-nodes-azure.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Joining Nodes via Azure Managed Identity
+description: Use the Azure join method to add Nodes to your Teleport cluster on Azure
+---
+
+This guide will explain how to use the **Azure join method** to configure
+Teleport Nodes and Proxy Service instances to join your Teleport cluster without
+sharing any secrets when they are running in an Azure Virtual Machine.
+
+The Azure join method is available in Teleport 12.1+. It is available to any
+Teleport Node or Proxy running in an Azure Virtual Machine. The Virtual Machine
+must have a
+[Managed Identity](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
+assigned to it with permission to read virtual machine info.
```

Comment:
> The Virtual Machine
must have a
[Managed Identity](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
assigned to it with permission to read virtual machine info.

I think we should move this sentence to the Prerequisites section so all prerequisites are listed in one place.